tests/AutoReview/ProjectCodeTest.php

```diff
@@ -389,6 +389,19 @@ public function testThereIsNoPregFunctionUsedDirectly(string $className): void
         self::assertNotContains('preg_split', $calledFunctions, $message);
     }
 
+    /**
+     * @dataProvider provideTestClassCases
+     *
+     * @param class-string $className
+     */
+    public function testThereIsNoUsageOfSetAccessible(string $className): void
+    {
+        $calledFunctions = $this->extractFunctionNamesCalledInClass($className);
+
+        $message = \sprintf('Class %s must not use "setAccessible()", use "Closure::bind()" instead.', $className);
```

Comment:
to be honest I don't see benefit of using one over another.

i'm equally sad with both approaches, they are hacks to rely on internals that shall not be messed with from outside.

Reply:
Isn't this similar to `array_map('trim', $a);` vs `array_map(fn ($x) => trim($x), $a);`?

Reply:
good reason, thank you 👍🏻